app/build.gradle

```diff
@@ -133,3 +154,19 @@ dependencies {
     debugImplementation "androidx.fragment:fragment-testing:$fragment_version"
 
 }
+
+sonarqube {
+    properties {
+        property "sonar.projectKey", "covid19risk_covidwatch-android"
+        property "sonar.organization", "covid19risk"
+        property "sonar.host.url", "https://sonarcloud.io"
+        property "sonar.coverage.jacoco.xmlReportPaths", "${buildDir}/reports/jacoco/debug/jacoco.xml,${buildDir}/reports/jacoco/release/jacoco.xml,${buildDir}/reports/jacocoCombined/debug/jacoco.xml"
+    }
+}
+//task jacocoTestReport(type: JacocoReport) {
```

Comment:
If added jacoco test report why then it's commented? Or this is covered with `com.vanniktech:gradle-android-junit-jacoco-plugin:0.16.0` plugin?

I general, I believe leaving commented code with the goal to uncomment it somewhen is redundant as we have git history and if you want to return some code just check it in the history and put it back.

Reply:
Completely agree! This was accidentally left in after switching to `com.vanniktech:gradle-android-junit-jacoco-plugin:0.16.0` plugin, good catch.